DERMS/FieldSimulator/PowerSimulator/Calculations/WindGeneratorProductionCalculation.cs

```diff
@@ -39,9 +39,9 @@ public override void Calculate(PowerGridSimulatorStorage powerGridSimulatorStora
 
         private float CalculateGeneratedPower(float windSpeed)
         {
-            if (windSpeed > startUpSpeed && windSpeed < nominalSpeed)
+            if (windSpeed >= startUpSpeed && windSpeed < nominalSpeed)
             {
-                return (windSpeed - startUpSpeed) * 0.035f * nominalPower;
+                return (windSpeed - startUpSpeed) * 0.035f;
```

Comment:
Na drajvu stvarno stoji tako, ali time ne možemo da uvažimo snagu. Ali je neophodna nominalna snaga kako bi mogao barem na jedan način da se uvede koeficijent jačine vetrogeneratora. Drugim rečim svi generatori će proizvoditi istu količinu aktivne snage kada je brzina vetra ispod njihove nominalne.

Reply:
Da, kapiram, u pravu si. Treba proveriti to.